omnibus/package-scripts/updater-rpm/prerm

```diff
@@ -2,10 +2,10 @@
 #
 # Perform necessary datadog-agent setup steps prior to remove the old package.
 
-readonly INSTALL_DIR=/opt/datadog/updater
+readonly INSTALL_DIR=/opt/datadog-packages/installer_boot
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Why the `_boot` suffix?

Reply:
added this one temporarily to quickly get out of /opt/datadog/
we'll have multiple versions in the dedicated installer package directory, we'll likely remove it then